client/lib/user/support-user-interop.js

```diff
@@ -112,7 +112,7 @@ const onTokenError = error => {
 /**
  * Inject the support user token into all following API calls
  */
-export const supportUserBoot = () => {
+export async function supportUserBoot() {
```

Comment:
These methods were already async, in that they triggered behaviour that waited behind a promise (namely, the Redux store dispatch). Marking them as `async` is an improvement, in my opinion, since it allows for a consumer to optionally wait for the full completion, rather than discarding the promises that are created within the method, as was being done before. It's functionally equivalent to returning the promise in the sync method, which would be another way of going about doing the same thing.

I will, however, make `bypassPersistentStorage` sync, as it doesn't need to be async considering our current implementation.